packages/components/src/components/checkbox-group/checkbox-group.stories.ts

```diff
@@ -1,16 +1,25 @@
 import '../../solid-components';
 import { html } from 'lit-html';
 import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
-import { userEvent } from '@storybook/test';
-import { waitUntil } from '@open-wc/testing-helpers';
 
 const { argTypes, parameters } = storybookDefaults('sd-checkbox-group');
 const { generateTemplate } = storybookTemplate('sd-checkbox-group');
 const { overrideArgs } = storybookHelpers('sd-checkbox-group');
 
+/**
+ * Used to select multiple options from a list of options.
+ *
+ * **Related templates:**
```

Comment:
Can we have related components 1st
2nd related templates please?

Reply:
Of couse! Updated :)